.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
We should probably add a littler more here (e.g. these weird mac files `.DS_STORE`)

Reply:
I don't have mac, could you add it? or passing me a list with these files?

Reply:
e.g. https://github.com/gnosis/safe-contracts/blob/development/.gitignore